mock-core-configs/etc/mock/amazonlinux-2023-x86_64.cfg

```diff
@@ -0,0 +1,5 @@
+include('templates/amazonlinux-2023.tpl')
+
+config_opts['root'] = 'amzn2023'
```

Comment:
Please change this to `amazonlinux-2023-x86_64`

Reply:
As above, fixing.